test/common/index.js

```diff
@@ -699,7 +696,7 @@ Object.defineProperty(exports, 'hasSmallICU', {
 });
 
 // Useful for testing expected internal/error objects
-exports.expectsError = function expectsError({code, type, message}) {
+function expectsError({code, type, message}) {
```

Comment:
Could you undo this?
Generally it's good, but it's just we have several other open `expectsError` PRs that will conflict
https://github.com/nodejs/node/pull/14089, https://github.com/nodejs/node/pull/14088, https://github.com/nodejs/node/pull/14058, https://github.com/nodejs/node/pull/13829, https://github.com/nodejs/node/pull/13686